From b83b0bb19da6023c64716b4f0554383ecdac12d9 Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Mon, 3 Dec 2012 07:54:59 +0200
Subject: [PATCH 0368/1825] KW40 Nand bug fix. Added translation of defconfigs

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit 6269476749173dd3eb8f42168fb842b278b33a7d

	1. CONFIG_MV_INCLUDE_PDMA to MV_INCLUDE_PDMA
	2. CONFIG_MTD_NAND_NFC_INIT_RESET to MTD_NAND_NFC_INIT_RESET

Change-Id: I13ee972472f00e1d255d749b3fe21841d16917d7
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Signed-off-by: Kosta Zertsekel <konszert@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/mach-armada370/config/mvSysNfcConfig.h |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-armada370/config/mvSysNfcConfig.h b/arch/arm/mach-armada370/config/mvSysNfcConfig.h
index a8365b64..72a9d7d 100644
--- a/arch/arm/mach-armada370/config/mvSysNfcConfig.h
+++ b/arch/arm/mach-armada370/config/mvSysNfcConfig.h
@@ -34,3 +34,11 @@ disclaimer.
 ** Base address for SPI registers.
 */
 #define MV_NFC_REGS_BASE		(MV_NFC_REGS_OFFSET)
+
+#ifdef CONFIG_MV_INCLUDE_PDMA
+#define MV_INCLUDE_PDMA
+#endif
+
+#ifdef CONFIG_MTD_NAND_NFC_INIT_RESET
+#define MTD_NAND_NFC_INIT_RESET
+#endif
-- 
1.7.5.4

